django/contrib/admin/templates/admin/filter.html

```diff
@@ -1,8 +1,19 @@
 {% load i18n %}
-<h3>{% blocktranslate with filter_title=title %} By {{ filter_title }} {% endblocktranslate %}</h3>
+
+{% with field=title.split|join:"_" %}
```

Comment:
I don't think that `title` is the best choice here. I'd rather add an `id` to a context in the `admin_list_filter`:
https://github.com/django/django/blob/0f6946495a8ec955b471ca1baaf408ceb53d4796/django/contrib/admin/templatetags/admin_list.py#L437-L444
maybe based on `lookup_kwarg` :thinking:

Reply:
Good idea! Done!